packages/generator/package.json

```diff
@@ -32,9 +32,10 @@
       "coverage",
       "test",
       "test_integration",
-      "gulpfile.ts",
-      "renderedTemplates",
-      "index.js"
+	  "renderedTemplates",
```

Comment:
I think we're doing this wrong...shouldn't we swap the exclude block for an include 'src'?

Reply:
makes sense